internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -548,16 +548,23 @@ func (provisioner *KopsProvisioner) ResizeCluster(cluster *model.Cluster, awsCli
 		return err
 	}
 
-	// TODO: Find out where to put this logic correctly
-	// if cluster.ProvisionerMetadataKops.RotatorRequest.Config != nil {
-	// 	if *cluster.ProvisionerMetadataKops.RotatorRequest.Config.UseRotator {
-	// 		logger.Info("Using node rotator for node resize")
-	// 		err = provisioner.RotateClusterNodes(cluster)
-	// 		if err != nil {
-	// 			return err
-	// 		}
-	// 	}
-	// }
+	requiresRollingUpdate, err := kops.RollingUpdateClusterRequired(kopsMetadata.Name)
+	if err != nil {
+		return err
+	}
+
+	if requiresRollingUpdate {
```

Comment:
nit: looks like this boolean `requiresRollingUpdate` should be something like `requiresClusterRotation`.  Because whether this value is `true` or `false`, we always do `RollingUpdateCluster`.

Reply:
I like that name more, yeah.